tests/testthat/setup-bugRzilla.R

```diff
@@ -12,10 +13,68 @@ if (!nzchar(Sys.getenv("R_BUGZILLA"))) {
 
 use_key()
 
+test_that("create_bugzilla_key works", {
+  vcr::use_cassette("create_bugzilla_key", {
+    host <- "https://bugs.r-project.org/bugzilla/"
+    cbk <- create_bugzilla_key(host)
+  })
+  expect_equal(a <- paste0(host, "userprefs.cgi?tab=apikey"), url)
+  cli::test_that_cli("cli_alert_warning works", {
+    expect_snapshot({
+      cli::cli_alert_warning("This package hasn't been tested with this Bugzilla version.")
+    })
+  })
```

Comment:
I didn't knew about the cli::test_that nice finding. However, what is being tested here? The snapshot doesn't seem to capture the call to `create_bugzilla_key`. But maybe I do not understand well how it should be used.

Reply:
it was not working while it was on the setup-bugRzilla.R file but It is working now. It has created an api.md this will test the cli test outputs.